server/server.ts

```diff
@@ -127,14 +126,17 @@ export function createServer(): Server {
       );
     });
     app.use(express.static(path.join(import.meta.dirname, '../frontend/dist')));
-    app.get('*', function (_req: any, res: { sendFile: (arg0: any) => void }) {
-      res.sendFile(
-        path.join(import.meta.dirname, '../frontend/dist', 'index.html')
-      );
-    });
+    app.get(
+      '/*splat',
```

Comment:
c'est quoi ?

Reply:
The wildcard * must have a name, matching the behavior of parameters :, use /*splat instead of /*